src/pubsub/index.js

```diff
@@ -589,7 +595,7 @@ class PubsubBaseProtocol extends EventEmitter {
     for (const topic of message.topicIDs) {
       const validatorFn = this.topicValidators.get(topic)
       if (!validatorFn) {
-        continue
+        continue // eslint-disable-line 
```

Comment:
you could just flip the logic here to `if (validatorFn) { await validatorFn(..) }` to avoid the disable

Reply:
I will also update this in the follow up PR